test/verify/machineslib.py

```diff
@@ -201,6 +201,15 @@ def tearDown(self):
         # which need to be adjusted before we can do that.
         pass
 
+    def addIface(self, name):
```

Comment:
That's something which I'd rather like to have in testlib.py, so that the networking tests can use it as well. It should then also generate a name instead of asking the caller to make up one. I. e. it should keep a global counter or a set of used names or similar. But I'm happy to do this as a follow-up. Added to #13717